### PR TITLE
feat(provider/ecs): Added DisableCluster stage

### PR DESCRIPTION
**Pull request by BrunoCarrier**
feat(provider/ecs): Added DisableCluster stage

@anotherchrisberry @icfantv @jrsquared @christopherthielen

### CODE REVIEW

**Comment by icfantv** [comment]
@BrunoCarrier hey Bruno, I’m no longer at Netflix.  Can you please stop tagging me in your PRs?  Thanks.

**Comment by BrunoCarrier** [comment]
Oh, I am sorry - yes I will stop tagging you :) Have a great day!

**Comment by icfantv** [comment]
No worries.  Thanks.

File: app/scripts/modules/ecs/pipeline/stages/disableCluster/ecsDisableClusterStage.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.disableClusterStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'disableCluster',
+      cloudProvider: 'ecs',
+      templateUrl: require('./disableClusterStage.html'),
+      executionConfigSections: ['disableClusterConfig', 'taskStatus'],
```
Comment (jrsquared) [inline app/scripts/modules/ecs/pipeline/stages/disableCluster/ecsDisableClusterStage.js]: shouldn't need this `executionConfigSections` either

Comment (BrunoCarrier) [reply app/scripts/modules/ecs/pipeline/stages/disableCluster/ecsDisableClusterStage.js]: Done

File: settings.js [hunk 2]
```diff
@@ -144,7 +144,7 @@ window.spinnakerSettings = {
     jobs: false,
     snapshots: false,
     travis: false,
-    pipelineTemplates: false,
+    pipelineTemplates: true,
```
Comment (jrsquared) [inline settings.js]: Not sure this should be a part of the PR

Comment (BrunoCarrier) [reply settings.js]: oh my, sorry for this :) I will correct it

Comment (BrunoCarrier) [reply settings.js]: Done

**Pull request merged**